common/components/deploy_model/spec.yaml

```
@@ -121,15 +121,15 @@ inputs:
 
   max_queue_wait_ms:
     type: integer
-    default: 500
+    default: 60000
```

Should probably be 90000 to match the script.
```suggestion
    default: 90000
```

---

Never mind. 60K is the default.